data/Language/JavaScript.yaml

```diff
@@ -0,0 +1,17 @@
+identifier: JavaScript
+wikidata: Q2005
+aliases:
+  - js
+  - ecmascript
+  - node
+extensions:
+  - js
```

Comment:
On coala repo we use `.js` as extension.

Reply:
@li-boxuan Please check now

Reply:
Hmm sorry, I just found out https://github.com/coala/coAST/blob/master/data/Language/Python.yaml also omits `.` @jayvdb What kind of format are we going to force on this repo?